release-notes/unreleased/NATIVE-632-WindowsOs-gradle-build-error-xcconfig.yml

```diff
@@ -0,0 +1,6 @@
+issue_key: NATIVE-623
```

Comment:
Imo a release note is not necessary and even does not make sense since it makes the release notes bigger without benefit.